### PR TITLE
this is wrong

### PR DESCRIPTION
**Pull request by farhanhaider1**
make a comment

### CODE REVIEW

**Review by farhanhaider1 — COMMENTED** [review]
what do you think?

File: index.html [hunk 1]
```diff
@@ -6,6 +6,6 @@
     <title>Document</title>
 </head>
 <body>
-    
+    <p>this is wrong</p>
```
Comment (Sammbacke) [inline index.html:9]: actually this is right

Comment (Tenzilla) [reply index.html:9]: This seems good

Comment (farhanhaider1) [reply index.html:9]: i want a laughing emoji

File: index.html [hunk 2]
```diff
@@ -6,6 +6,6 @@
     <title>Document</title>
```
Comment (Tenzilla) [inline index.html:6]: Testing: Title name "Document" is nice

File: index.html (hunk 1 above)
Comment (Tenzilla) [reply index.html:9]: yeah i only see other ones :((

**Review by Tenzilla — CHANGES_REQUESTED** [review]

**Review by Tenzilla — APPROVED** [review]

**Pull request closed without merging**